src/docs/product/sentry-basics/tracing/event-detail.mdx

```diff
@@ -62,16 +62,20 @@ The span view may be limited to one project at a time if you're on the [Team pla
 
 </Alert>
 
-#### Traversing Transactions
+#### Embedding and Traversing Transactions
 
-Some spans within a transaction may be the parent of another transaction. Under these circumstances, some Span IDs will have a "View Child" or "View Children" button. These will potentially lead to another transaction or a list of transactions.
+In some cases, a span may be the parent of one or more transactions. You can identify which span has children with the expand callout. Clicking this icon will embed the first child transaction. When a transaction is embedded, the collapse callout will appear for you to hide it again.
```

Comment:
I've also made some other edits here based on what the UI seems to be doing, but please confirm. "Callout" seems like a pretty UI/UX-specific term. Suggest sticking with icon. 

```suggestion
In some cases, a span may be the parent of one or more transactions. Spans that have children are identified by the expand icon. Clicking this icon will display the first embedded child transaction. When that transaction is displayed, the collapse icon will appear so that you can the transaction again.
```